src/GeometryInstancing/Shaders/Vertex.glsl

```diff
@@ -0,0 +1,30 @@
+#version 330 core
+
+struct Camera_ProjView
+{
+    mat4 View;
+    mat4 Proj;
+};
+
+layout(std140) uniform projView
+{
+    Camera_ProjView field_projView;
+};
+
+layout(location = 0)in vec2 Position;
```

Comment:
These layout qualifiers actually have no effect on Veldrid. It might be better to just remove them so it doesn't mislead anyone.

BTW: I want to support using `layout` specifiers, but I think it needs to be an opt-in thing as otherwise it's a breaking change.